movie-browser/src/components/MovieList.js

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+
+const URL = 'https://ghibliapi.herokuapp.com/films';
+
+class MovieList extends Component {
+  state = {
+    movies: [],
+  };
+
+  componentDidMount() {
+    axios
```

Comment:
api call shouldn't be on this component. Regarding SEPARATION OF CONCERNS this should be a MovieList, so it should only display the movies, and it shouldn't be concerned with HOW you fetch them. So, the fetch should occur on its parent.

Reply:
moved logic and state to App.j and using MovieList.js just to render the movies